bitcoin/src/psbt/map/input.rs

```diff
@@ -136,6 +136,15 @@ pub struct Input {
 /// type can be either [`EcdsaSighashType`] or [`TapSighashType`] but it is not possible to know
 /// directly which signature hash type the user is dealing with. Therefore, the user is responsible
 /// for converting to/from [`PsbtSighashType`] from/to the desired signature hash type they need.
+///
+/// # Examples
+///
+/// ```
+/// use bitcoin::{EcdsaSighashType, TapSighashType};
+///
+/// let ecdsa_sighash_all: EcdsaSighashType = EcdsaSighashType::All.into();
+/// let tap_sighash_all: TapSighashType = TapSighashType::All.into();
```

Comment:
This has a typo in the annotations.
```rust
/// use bitcoin::{EcdsaSighashType, psbt::PsbtSighashType, TapSighashType};
///
/// let ecdsa_sighash_all: PsbtSighashType = EcdsaSighashType::All.into();
/// let tap_sighash_all: PsbtSighashType = TapSighashType::All.into();
```

If clippy linted examples this would have been caught.

Reply:
Great catch! Done in 4f0eafa2b14e7013213cd2dc4a232687bc746645